maintainers/maintainer-list.nix

```diff
@@ -118,6 +118,12 @@
     githubId = 2258953;
     name = "Aaron Schif";
   };
+  aaschmid = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add aaschmid`` and sort it alphabetically.

Reply:
It is a separate commit already and sorted alphabetically. I can adjust the commit message, though.